test/message/nexttick_throw.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
Yeah, I think it's better that way, less noisy comments all over the tests, will work something out.